spec/lib/authorities/mysql_table_based_authority_spec.rb

```diff
@@ -8,6 +8,16 @@
     Qa::Authorities::Local.register_subauthority('language', described_class.to_s)
   end
 
+  describe "::table_name" do
```

Comment:
Shorthand for class method? Instance methods are proceeded by #, or at least that's what I see in the Ruby docs.